explainer.md

```diff
@@ -609,10 +622,23 @@ P1 and P2 are for the same user.
 
 For many current online payment flows this may not be a significant concern, as
 the user already provides sufficient information to do this joining anyway
-(e.g. their address), however it could become a privacy attack if, e.g.,
-payment tokenization becomes commonplace.
+(e.g. their address) However, if payment methods that involve less identifying
+information (e.g., tokenization) become commonplace, it is important that
+ecosystem stakeholders take steps to preserve user privacy. For example:
+
+- Payment systems might establish rules that place limits on storage of
+  credential ID(s) by third parties.
+
+- When a Relying Party assigns multiple instruments to a single SPC credential,
+  it might choose not to share that credential ID with other parties. In this
+  case, the Relying Party could still use the SPC credential itself (in either a
+  first-party or third-party context) to authenticate the user.
+
+- A Relying Party (e.g., a bank) might enable the user to register a distinct
+  SPC credential per payment instrument. This would not prevent the Relying
+  Party from joining those accounts internally.
 
-For potential ways to defeat such an attack, [see issue
+For potential future spec changes to defeat such an attack, [see issue
```

Comment:
Suggest deleting the para "For potential future spec changes" since we closed 77. Otherwise good to go!

Reply:
Done.